pyo3-macros-backend/src/pyclass.rs

```diff
@@ -2320,7 +2344,10 @@ fn pyclass_class_getitem(
 /// and attributes of `#[pyclass]`, and docstrings.
 /// Therefore it doesn't implement traits that depends on struct fields and enum variants.
 struct PyClassImplsBuilder<'a> {
-    cls: &'a syn::Ident,
+    /// Identifier of the class Rust struct
+    cls_ident: &'a Ident,
+    /// Name of the class in Python
+    cls_name: &'a Ident,
```

Comment:
We need the two "flavors" because for complex enum variants the first one will be `Enum_Variant` and the second one just `Variant`